packages/config/src/android/Updates.ts

```diff
@@ -99,3 +103,73 @@ export function setVersionsConfig(config: ExpoConfig, manifestDocument: Document
 
   return manifestDocument;
 }
+
+export function formatApplyLineForBuildGradle(projectDir: string, config: ExpoConfig): string {
+  const updatesGradleScriptPath = projectHasModule(
+    'expo-updates/scripts/create-manifest-android.gradle',
+    projectDir,
+    config
+  );
+
+  if (!updatesGradleScriptPath) {
+    throw new Error(
+      "Could not find the build script for Android. This could happen in case of outdated 'node_modules'. Run 'npm install' to make sure that it's up-to-date."
```

Comment:
will this fail if a user attempts to eject or re-eject without expo-updates installed?

Reply:
it's not used in eject (it's not necessary there because those changes are already in the template), this code is intended to run on vanilla react-native projects.
Currently, the code that is executing it is responsible for checking if expo-updates is installed.